probe-rs/src/architecture/arm/dp/mod.rs

```diff
@@ -484,6 +533,72 @@ impl Register for TARGETID {
     const NAME: &'static str = "TARGETID";
 }
 
+bitfield! {
+    /// BASEPTR0, Initiale system address for the first component in the system (see ADI v6.0 B2.2.2)
+    #[derive(Clone)]
+    pub struct BASEPTR0(u32);
+    impl Debug;
+    /// Pointer least significant bits (bits [31:12] of the full address).
```

Comment:
Where is the remaining 12 bits?

Reply:
The 11bits in [11;1] are not used. 0 indicates if the whole ptr is valid.

Reply:
Ah, so I guess this can only point to higher addresses then :)

Reply:
Yes, on ADIv6 the AP live in a memory space.
- The BASEPTR sets the bits [63:12]
- The Register address sets the bits [12:2]
- All read/write operations are 4bytes aligned so the bits [1:0] are ignored.